spec/unit/context_spec.rb

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright:: Copyright (c) Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "spec_helper"
+require "chef/context"
+require "openssl"
+
+describe Chef::Context do
+  describe "when executed normally" do
+    before(:each) do
+      described_class.send(:reset_context)
+    end
+
+    it "#test_kitchen_context? should return false" do
+      expect(described_class.test_kitchen_context?).to be_falsey
+    end
+
+    it "#context_secret should be empty" do
+      expect(described_class.send(:fetch_env_value)).to be_falsey
+    end
+  end
+
+  context "when executed from test kitchen" do
+    before(:each) do
+      described_class.send(:reset_context)
+      allow(ENV).to receive(:fetch).with("IS_KITCHEN", false).and_return("true")
+    end
+
+    it "#context_secret should return the context key" do
```

Comment:
There is no more #context_secret anymore, correct?

Reply:
Oops, forgot to update the spec definition. Updated.